src/main/java/com/alkemy/ong/ports/output/email/SendGridEmailService.java

```diff
@@ -0,0 +1,58 @@
+package com.alkemy.ong.ports.output.email;
+
+import com.sendgrid.Method;
+import com.sendgrid.Request;
+import com.sendgrid.Response;
+import com.sendgrid.SendGrid;
+import com.sendgrid.helpers.mail.Mail;
+import com.sendgrid.helpers.mail.objects.Content;
+import com.sendgrid.helpers.mail.objects.Email;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import java.io.IOException;
+
+@Service
+@Slf4j
+public class SendGridEmailService implements EmailService {
+
+    private final SendGrid sendGridClient;
+
+
+    public SendGridEmailService(SendGrid sendGridClient) {
+        this.sendGridClient = sendGridClient;
+
+    }
+
+    @Override
+
+    public void sendText(String from, String subject, String body) {
+        Response response = sendEmail(from,  subject, new Content("text/plain", body));
+        log.info("Status Code: " + response.getStatusCode() + ", Body: " + response.getBody() + ", Headers: "
+                + response.getHeaders());
+
+    }
+
+    @Override
+    public void sendHTML(String from,  String subject, String body) {
+        Response response = sendEmail(from, subject, new Content("text/html", body));
+        log.info("Status Code: " + response.getStatusCode() + ", Body: " + response.getBody() + ", Headers: "
+                + response.getHeaders());
+    }
+
+    private Response sendEmail(String from,  String subject, Content content){
+        Mail mail = new Mail(new Email(from), subject, new Email(), content);
```

Comment:
Acá le estas pasando el email `from`, y el email to vacío. Por favor pasar el email from que es siempre el mismo desde la variable en `email.from` y el email to es el que te pasan por parámetro. ¿Cómo está funcionando esto?